_data/content/projects/en.yml

```diff
@@ -21,212 +21,60 @@ page_data:
       img: ":project2_thumb.jpg"
       img_title: "img title2"
-      date: "2021-04-23"
+      date: "2023-05-10"
       post: |
-        # Title
-        This is project content.
-
-        ![Image](:project2_thumb.jpg)
+        # El cine también se lee (Cinema can also be read) exhibition- Castilla La Mancha Library, Toledo, Spain
 
     # quote
     - type: id_quote
-      project_name: "Example Project"
-      project_excerpt: "William Shakespeare"
+      project_name: "‘Nuevas expografías: entre la tecnología y las experiencias’ seminar"
+      project_excerpt: "Fundación Mainel"
       #img: ":project1_thumb.jpg"
       #img_title: "img title3"
-      date: "2021-05-27"
+      date: "2023-05-10"
       post: |
-        Let me not to the marriage of true minds
-        Admit impediments. Love is not love
-        Which alters when it alteration finds,
-        Or bends with the remover to remove.
-        O no, it is an ever-fixed mark
-        That looks on tempests and is never shaken;
-        It is the star to every wand'ring barque,
-        Whose worth's unknown, although his height be taken.
-        Love's not Time's fool, though rosy lips and cheeks
-        Within his bending sickle's compass come;
-        Love alters not with his brief hours and weeks,
-        But bears it out even to the edge of doom.
-        If this be error and upon me proved,
-        I never writ, nor no man ever loved.
+        # ‘Nuevas expografías: entre la tecnología y las experiencias’ seminar in Coloquios Cultura Visual Contemporaneo series- Fundación Mainel, Valencia, Spain
+
     - type: id_quote
-      project_name: "Example Project"
-      project_excerpt: "Albert Einstein"
+      project_name: "Rise-Rosa-Rage Archive of Socio-Political Graphics, Rome, Italy / Online"
+      project_excerpt: "Rise Rosa Rage Archive"
       img: ":project2_thumb.jpg"
       img_title: "img title4"
-      date: "2021-06-08"
+      date: "2023-05-10"
       post: |
-        Two things are infinite: the universe and human stupidity; and I'm not sure about the universe.
+        # Rise-Rosa-Rage Archive of Socio-Political Graphics, Rome, Italy / Online
+
     - type: id_quote
-      project_name: "Example Project"
-      project_excerpt: "Mae West"
+      project_name: "Module on MA Course Barreira Higher School"
+      project_excerpt: "Barreira Higher School of Art and Design"
       img: ":project1_thumb.jpg"
       img_title: "img title5"
-      date: "2021-08-20"
-      post: |
-        You only live once, but if you do it right, once is enough.
-    - type: id_quote
-      project_name: "Example Project"
-      project_excerpt: "Mahatma Gandhi "
-      img: ":project2_thumb.jpg"
-      img_title: "img title6"
-      date: "2021-12-20"
+      date: "2023-05-10"
       post: |
-        Be the change that you wish to see in the world.
+        # Module on MA Course Barreira Higher School of Art and Design, Valencia, Spain
+        
```

Comment:
Last line is not empty, has some spaces:
![image](https://github.com/juanjoseortega-aircury/cc-test/assets/105167930/2e5c4d5d-0177-4a49-9bb5-bf3c547c6e46)